netstandard/tools/HandlePackageFileConflicts.cs

```diff
@@ -155,6 +241,75 @@ private int GetPackageRank(ITaskItem item)
             return rank;
         }
 
+        /// <summary>
+        /// Get's the key to use for identifying reference conflicts
+        /// </summary>
+        /// <param name="item"></param>
+        /// <returns></returns>
+        private static string GetReferenceKey(ITaskItem item)
+        {
+            var aliases = item.GetMetadata("Aliases");
+
+            if (!String.IsNullOrEmpty(aliases))
+            {
+                // skip conflict detection for aliased assemblies
+                return null;
+            }
+
+            // We only handle references that have path information since we're only concerned
+            // with resolving conflicts between file references.  If conflicts exist between 
+            // named references that are found from AssemblySearchPaths we'll leave those to
+            // RAR to handle or not as it sees fit.
+            var sourcePath = GetSourcePath(item);
+
+            if (String.IsNullOrEmpty(sourcePath))
+            {
+                return null;
+            }
+
+            // The following is similar to Path.GetFileName but does not throw and returns null
+            // rather than empty for a directory.
+            var length = sourcePath.Length;
+            for (int i = length - 1; --i >= 0;)
```

Comment:
Ok, that's fair, but I hate catching exceptions and we cannot be a failure point here.